js/views/baseVw.js

```diff
@@ -61,7 +62,7 @@ export default class baseVw extends View {
       const $field = $(field);
       const varType = $field.data('var-type');
 
-      let val = $field.val();
+      let val = sanitizeHtml($field.val());
```

Comment:
I think it's safer the sanitization happens here:

```
      } else {
        data[name] = val;
      }
```

Doing it right off the bat is concerning to me because there are subsequent checks on the value that may not be compatible with what the sanitization is changing the value to.